docs/v1/P4Runtime-Spec.adoc

```diff
@@ -2312,12 +2150,12 @@ modify objects in the data plane via an `Update` message.  For each, a
 description is given of the parts of that entity that are data plane
 volatile.
 
-#### ExternEntry
+==== ExternEntry
```

Comment:
ExternEntry should be one level deeper, i.e. `=====`

Reply:
Note:  I checked all others in the generated table of contents, and except for this one heading, they all look correct!  Yay!